packages/config/__tests__/__resources__/ProfInfoApp_home_team_config_proj/ProfInfoApp.schema.json

```diff
@@ -0,0 +1,393 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$version": 0.4,
```

Comment:
Not sure what order we plan to merge PRs in, but we should update this version to `"1.0"`.

Reply:
if we merge #778 before this one (which I would advise we do), then I don't mind updating it here 😋